src/TestSuite/Fixture/TestFixture.php

```diff
@@ -439,6 +439,16 @@ public function truncate(ConnectionInterface $connection): bool
         return true;
     }
 
+    /**
+     * Returns whether the fixture should be created/destroyed.
+     *
+     * @return bool
+     */
+    public function allowCreate(): bool
```

Comment:
I don't know about this method name. Would something like `hasManagedSchema()` work?

Reply:
Derp, just saw your comment. Of the options you had `isManaged()` fits better. I'm not a fan of `allowCreate()` as fixtures can create records & schema and the method name isn't clear to me.

Reply:
I didn't like it either, but wanted feedback before switching back.